### PR TITLE
adds destination docs for MetricStory

### PR DESCRIPTION
**Pull request by randallkanna**
### Proposed changes

Adds the MetricStory documentation for MetricStory.ai

### Merge timing
ASAP - This is ready to go

### Related issues (optional)

### CODE REVIEW

File: src/connections/destinations/catalog/metricstory/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+MetricStory supports the following methods, as specified in the Segment Spec.
+
+Page
+Send Page calls to ADD WHAT PAGE CALLS ARE USED FOR HERE. For example:
+
+analytics.page()
+Segment sends Page calls to MetricStory as a pageview.
+
+Identify
+Send Identify calls to ADD WHAT IDENTIFY CALLS ARE USED FOR HERE. For example:
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/metricstory/index.md:26]: Actually clarify how identify calls are interpreted by Metric story here. "ADD WHAT IDENTIFY CALLS ARE USED FOR HERE" is from the template

File: src/connections/destinations/catalog/metricstory/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+analytics.identify('userId123', {
+  email: 'john.doe@example.com'
+});
+Segment sends Identify calls to MetricStory as an identify event.
+
+Track
+Send Track calls to ADD WHAT Track CALLS ARE USED FOR HERE. For example:
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/metricstory/index.md:34]: Actually clarify how track calls are interpreted by Metric story here. "ADD WHAT TRACK CALLS ARE USED FOR HERE" is from the template

File: src/connections/destinations/catalog/metricstory/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Identify
+Send Identify calls to ADD WHAT IDENTIFY CALLS ARE USED FOR HERE. For example:
+
+analytics.identify('userId123', {
+  email: 'john.doe@example.com'
+});
+Segment sends Identify calls to MetricStory as an identify event.
+
+Track
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/metricstory/index.md:33]:
```suggestion
### Track
```

File: src/connections/destinations/catalog/metricstory/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+analytics.page()
+Segment sends Page calls to MetricStory as a pageview.
+
+Identify
+Send Identify calls to ADD WHAT IDENTIFY CALLS ARE USED FOR HERE. For example:
+
+analytics.identify('userId123', {
+  email: 'john.doe@example.com'
+});
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/metricstory/index.md:30]:
```suggestion
```js
analytics.identify('userId123', {
  email: 'john.doe@example.com'
});
```
```

File: src/connections/destinations/catalog/metricstory/index.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+Segment sends Identify calls to MetricStory as an identify event.
+
+Track
+Send Track calls to ADD WHAT Track CALLS ARE USED FOR HERE. For example:
+
+analytics.track('Login Button Clicked')
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/metricstory/index.md:36]:
```suggestion
```js
analytics.track('Login Button Clicked')
```

```

**Review by tcgilbert — APPROVED** [review]

**Review by pwseg — APPROVED** [review]

**Pull request merged**